pkg/statsd/parser.go

```diff
@@ -76,15 +76,20 @@ func (dp *DatagramParser) Run(ctx context.Context) {
 		case <-ctx.Done():
 			return
 		case dgs := <-dp.in:
-			accumM, accumE, accumB := uint64(0), uint64(0), uint64(0)
+			var metrics []*gostatsd.Metric
+
+			accumB, accumE := uint64(0), uint64(0)
 			for _, dg := range dgs {
-				m, e, b := dp.handleDatagram(ctx, dg.Timestamp, dg.IP, dg.Msg)
+				parsedMetrics, e, b := dp.handleDatagram(ctx, dg.Timestamp, dg.IP, dg.Msg)
```

Comment:
Mixed variable styles.
But I agree verbose name is better here. I have to look signature of handleDatagram to know what m, e, b means. And long names are used even in that function.

Reply:
Well I do like my verbose names, I'll update that.